src/mavlinktelemetry.cpp

```diff
@@ -474,14 +487,12 @@ void MavlinkTelemetry::onProcessMavlinkMessage(mavlink_message_t msg) {
             mavlink_msg_home_position_decode(&msg, &home_position);
             OpenHD::instance()->set_homelat((double)home_position.latitude / 10000000.0);
             OpenHD::instance()->set_homelon((double)home_position.longitude / 10000000.0);
-            LocalMessage::instance()->showMessage("Home Position set by OpenHD", 2);
+            //LocalMessage::instance()->showMessage("Home Position set by Telemetry", 7);
             break;
```

Comment:
Should this be un-commented so that it displays to the user?

Reply:
I was afraid of some FC sending this constantly as we do not have via mavlink requests yet. I noticed inav was sending gps_global_origin this way constantly. But I guess if nobody complain yet, we could uncomment.